### ISSUE TITLE
depth map

### ISSUE THREAD

**Issue by 04lm40** [AaltoVision/warped-skipconnection-nvs#2]
Hello, I would like to ask how to save the depth map, I have added the following content in the code, but the image is all black. Thank you!

img_numpy = util.tensor2depthim(self.depth_b.detach())
image_pil = Image.fromarray(img_numpy)
image_pil.save(source_name)
